stream-chat-android-ui-components/src/main/res/values/attrs_message_list_view.xml

```diff
@@ -397,5 +397,8 @@
         <attr name="streamUiPinnedMessageIndicatorIcon" format="reference" />
 
         <attr name="streamUiPinnedMessageBackgroundColor" format="color" />
+
+        <attr name="streamUiMessageStartMargin" format="dimension|reference" />
+        <attr name="streamUiMessageEndMargin" format="dimension|reference" />
```

Comment:
Out of curiosity, what does the message list look like when these values are:
- 0, but the predicate says avatars should be shown
- Half the default value, and predicate says avatars should show
- More than the default value

Reply:
The idea is not to prevent the user from creating a margin that doesn't look good, but allowing him/her to customize it the way they prefer.

In the first case they would need to set a margin in the bottom to avoid text and avatar overlap. Same goes for the second. The third works fine, but they may end up with too little space in the screen for actual text